submissions/f3d0t/js-oop/app.js

```diff
@@ -18,9 +18,8 @@ const playingAreaInfo = {
 	width: cellInfo.width * 4,
 	height: cellInfo.height * 5,
```

Comment:
You have access to `ctx` variable from engine.js.  If you have access to dynamically calculated value - use it instead of this ones.
```
/* Assign the canvas' context object to the global variable (the window
   * object when run in a browser) so that developers can use it more easily
   * from within their app.js files.
   */
  global.ctx = ctx;
  ```

Reply:
I see that window.ctx returns undefined..